.github/workflows/ci.yml

```diff
@@ -1,4 +1,4 @@
-name: Test
+name: Rust SDK
```

Comment:
This also handles testing the core and (currently) fastly, so it's not "rust sdk"

Reply:
I see the discussion in the other thread. I think it's OK to merge but we should address this later